pyproject.toml

```
@@ -30,7 +31,9 @@ build-backend = "poetry.core.masonry.api"
 [tool.pytest.ini_options]
 testpaths = [
     "tests/unit",
-    "tests/integration"
+    "tests/integration",
+    "integration: marks test as integration tests (select with '-m integration')"
+
```

This looks like a bad conflict resolution, you can delete it
```suggestion
    "tests/integration"
```